BZ1867750: Replace the "ssh core@" command to shut down nodes (#36290)
openshift/openshift-docs: modules/graceful-shutdown.adoc

----
<1> To ensure that the cluster can restart gracefully, plan to restart it on or before the specified date. As the cluster restarts, the process might require you to manually approve the pending certificate signing requests (CSRs) to recover kubelet certificates.

> . Shut down all of the nodes in the cluster. You can do this from your cloud provider's web console, or you can use the below command:

Reviewer: ```suggestion
. Shut down all of the nodes in the cluster. You can do this from your cloud provider's web console, or run the following loop:
```

Per the IBM style guide, you cannot use "below". Since this is a loop, my suggestion is above.

Author: Done

+
[source,terminal]
----

> $ for node in $(oc get nodes -o jsonpath='{.items[*].metadata.name}'); do oc debug node/${node} -- chroot /host shutdown -h 1; done

Author: A $ is not needed in front of the `oc`.  If given the for loop will not work.

----
+
.Example output
----
Starting pod/ip-10-0-130-169us-east-2computeinternal-debug ...
To use host binaries, run `chroot /host`
Shutdown scheduled for Mon 2021-09-13 09:36:17 UTC, use 'shutdown -c' to cancel.

Removing debug pod ...
Starting pod/ip-10-0-150-116us-east-2computeinternal-debug ...
To use host binaries, run `chroot /host`
Shutdown scheduled for Mon 2021-09-13 09:36:29 UTC, use 'shutdown -c' to cancel.
----

> 

Reviewer 2: You need to restore these + signs. They make the comment and note render correctly under the step. (Check the preview - the step numbers have reset after your change.)

```suggestion
+
```

Shutting down the nodes using one of these methods allows pods to terminate gracefully, which reduces the chance for data corruption.

> 

Reviewer 2: ```suggestion
+
```
